tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -183,7 +183,7 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
 
             StatsComputationEnabledInternal = config
                                      .WithKeys(ConfigurationKeys.StatsComputationEnabled)
-                                     .AsBool(defaultValue: false);
+                                     .AsBool(defaultValue: (ServerlessMiniAgent.IsAzureFunction || ServerlessMiniAgent.IsGCPFunction) ? true : false);
```

Comment:
[code style]
`<expression> ? true : false` can be simplified to `<expression>`

```suggestion
                                     .AsBool(defaultValue: ServerlessMiniAgent.IsAzureFunction || ServerlessMiniAgent.IsGCPFunction);
```

Reply:
fixed